Assets/Scripts/Game/UserInterfaceWindows/DaggerfallInventoryWindow.cs

```diff
@@ -1042,6 +1046,38 @@ protected virtual void LoadTextures()
 
         }
 
+        protected virtual void UnloadTextures()
```

Comment:
I've not been active on the project for a while, but if I remember correctly the Inventory window (plus a few others) essentially use singletons that are initialised each time they're shown, which I don't think re-creates the texture data being destroyed here. If you destroy the textures then they will need recreating every time the window is show. (a lot in most playing I think) I also can't see what's calling this method, but that's probably due to not diving deep enough into the PR.

Reply:
That's correct but this is only used by the trade window which inherits from the inventory window. Unlike its base class the trade window isn't persistent so it has to be cleaned up when it's popped.

Reply:
Ah, very good. Makes sense now.